src/plothist/__init__.py

```diff
@@ -116,3 +116,11 @@
             stacklevel=3,
         )
         break
+
+# Check version of boost_histogram
+import boost_histogram as bh
+
+if tuple(int(part) for part in bh.__version__.split(".")) < (1, 4, 0):
+    raise ImportError(
+        "The version of boost_histogram is lower than 1.4.0. Please update to the latest version to avoid issues (pip install --upgrade boost_histogram==1.4.0).",
```

Comment:
Remove "==1.4.0", I think we aim at always be up-to-date with the latest version of bh, so potentially >1.4.0